src/Tomcat/TomcatModel.ts

```diff
@@ -33,31 +33,50 @@ export class TomcatModel {
     public async saveServerList(): Promise<void> {
         try {
             await fse.outputJson(this._serversJsonFile, this._serverList.map((s: TomcatServer) => {
-                return { _name: s.getName(), _installPath: s.getInstallPath(), _storagePath: s.getStoragePath() };
+                return { _name: s.getName(), _installPath: s.getInstallPath(), _storagePath: s.getStoragePath(), _runInPlace: s.isRunInPlace() };
             }));
             vscode.commands.executeCommand('tomcat.tree.refresh');
         } catch (err) {
             console.error(err.toString());
         }
     }
 
-    public async updateJVMOptions(serverName: string) : Promise<void> {
-        const server: TomcatServer = this.getTomcatServer(serverName);
+    public async updateJVMOptions(server: TomcatServer) : Promise<void> {
```

Comment:
This method didn't semantically change much. Just sent portions of it to separate functions so that the branching changes I implemented were clearer to maintain.